src/Components/test/E2ETest/Tests/SignalRClientTest.cs

```diff
@@ -40,8 +41,9 @@ protected override void InitializeAsyncCore()
             Browser.Exists(By.Id("signalr-client"));
         }
 
+        public override Task InitializeAsync() => base.InitializeAsync(Guid.NewGuid().ToString());
+
         [Fact]
-        [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/27156")]
         public void SignalRClientWorksWithLongPolling()
```

Comment:
```suggestion
        [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/27156")]
        public void SignalRClientWorksWithLongPolling()
```

Reply:
?

Reply:
We aren't removing the `QuarantinedTest` attribute for fixed tests now, so just added it back in. PR originally created prior to your email of changes to the quarantine process.

Reply:
Updated PR title to reflect that we're just fixing the test, not unquarantining it.